editor.planx.uk/src/lib/feedback.ts

```diff
@@ -25,10 +25,9 @@ export async function getInternalFeedbackMetadata(): Promise<FeedbackMetadata> {
     breadcrumbs,
     currentCard: node,
     computePassport,
-    fetchCurrentTeam,
+    teamId,
```

Comment:
This appears to be `undefined`, so I don't actually think this solution works and we need to do the fetch as before.

On the pizza, I determined this by: 
- Noticing the Feedback Log was empty, even though the front end interaction appeared to succeed (`teamId` is required to properly filter/populate view)
- Inspecting the graphql mutation `InsertFeedback` in my network tab - you'll see it's missing a `teamId` variable (graphql omits variables if undefined)
- double-checking `window.api.getState().teamId` in the console, which also returned `undefined` when on a /published link

Reply:
@jessicamcinchak using the updated `getBySlug()` with `submission_email` removed seems to fix this for me locally, with team_id being in the database.

Going to check it on the pizza now

Reply:
Pizza is working for me